### PR TITLE
Add listing replicas

### PR DESCRIPTION
**Pull request by wagoodman**
This change allows for grype-db listing files to be replicated to multiple S3 locations via grype-db-manager application configuration:
```yaml
listing-file-name: listing.json
s3-path: grype/databases
s3-bucket: my-awesome-bucket.io
aws-region: us-west-2
download-url-prefix: https://my-awesome-bucket.io

# this maps to feed service API version paths
listing_replicas:
  - s3-path: v1/grype/databases
  - s3-path: v2/grype/databases
    s3-bucket: another-bucket.io

```

### CODE REVIEW

File: manager/src/grype_db_manager/grypedb.py [hunk 1]
```diff
@@ -462,11 +462,9 @@ def run(self, *args, provider_root_dir: str, config: str) -> int:
 
         env = dict(  # noqa: PIE804
             **os.environ.copy(),
-            **{
-                "GRYPE_DB_VUNNEL_ROOT": provider_root_dir,
-                "GRYPE_DB_CONFIG": config,
-                "GRYPE_DB_LOG_LEVEL": level,
-            },
+            GRYPE_DB_VUNNEL_ROOT=provider_root_dir,
```
Comment (willmurphyscode) [inline manager/src/grype_db_manager/grypedb.py:465]: I assume this is an unrelated change just making the Python nicer?

**Review by willmurphyscode — APPROVED** [review]

File: manager/src/grype_db_manager/grypedb.py (hunk 1 above)
Comment (wagoodman) [reply manager/src/grype_db_manager/grypedb.py:465]: ruff did this -- I like it
